components/settings/Account/Usage.tsx

```diff
@@ -0,0 +1,102 @@
+import { UsageByService } from "@3rdweb-sdk/react/hooks/useApi";
+import { SimpleGrid, Spinner, VStack } from "@chakra-ui/react";
+import { UsageCard } from "./UsageCard";
+import { useMemo } from "react";
+import { Heading } from "tw-components";
+
+interface UsageProps {
+  usage: UsageByService | undefined;
+  usageLoading: boolean;
+}
+
+export const Usage: React.FC<UsageProps> = ({ usage, usageLoading }) => {
+  const bundlerTotal = useMemo(() => {
+    if (!usage?.bundler || usage.bundler.length === 0) {
+      return <i>N/A</i>;
+    }
+    const total = usage.bundler?.[0].billableUsd || 0;
```

Comment:
Why are we grabbing the first element here?

Reply:
yeah long story, but the format of usage is weirdly delivered from API now, per chain, but total for each chain. we need to fix API structure when we get to other services.